utils/CacheManager.py

```diff
@@ -40,16 +42,16 @@ def format_json_after_fetch(self, string):
         return ast.literal_eval(string)
 
     def init_db(self):
-        # try:
-        c = self.database.cursor()
+        try:
+            c = self.database.cursor()
 
-        c.execute("CREATE TABLE IF NOT EXISTS local_cache (url TEXT NOT NULL PRIMARY KEY, name TEXT, schema_hash TEXT, schema TEXT, img_hash TEXT, img BLOB)")
+            c.execute("CREATE TABLE IF NOT EXISTS local_cache (url TEXT NOT NULL PRIMARY KEY, name TEXT, schema_hash TEXT, schema TEXT, img_hash TEXT, img BLOB)")
```

Comment:
The cache schema should look like this
CREATE TABLE IF NOT EXISTS local_cache (key TEXT NOT NULL PRIMARY KEY, value BLOB)
There should be no schema, or image references.